<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-36D5C16" ignored="0" name="Command line test" tests="3" failures="1" timestamp="2022-05-21T14:19:12 MST" time="8.04" errors="0">
  <testcase name="testAppleTitle" time="0.99" classname="scripts._01_AppleTest"/>
  <testcase name="testAppleURL" time="0.873" classname="scripts._01_AppleTest"/>
  <testcase name="x" time="0.0" classname="scripts._01_AppleTest">
    <failure type="java.lang.AssertionError" message="Failure on purpose - bug ticket created KJKJ798798">
      <![CDATA[java.lang.AssertionError: Failure on purpose - bug ticket created KJKJ798798
at scripts._01_AppleTest.x(_01_AppleTest.java:55)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- x -->
</testsuite> <!-- Command line test -->
